promql/functions.go

```diff
@@ -212,18 +214,18 @@ func funcDropCommonLabels(ev *evaluator, args Expressions) Value {
 		}
 	}
 
-	for _, el := range vector {
+	for _, el := range vec {
 		for k := range el.Metric.Metric {
 			if _, ok := common[k]; ok {
 				el.Metric.Del(k)
 			}
 		}
 	}
-	return vector
+	return vec
 }
 
-// === round(vector ExprVector, toNearest=1 Scalar) Vector ===
-func funcRound(ev *evaluator, args Expressions) Value {
+// === round(vector model.ValVector, toNearest=1 Scalar) Vector ===
```

Comment:
I guess `Scalar` would be `model.ValScalar` now here. There's also some other functions where this isn't consistent.